pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
+    slow: marks tests that are slow (deselect with '-m "not slow"')
+    crmpdb: marks tests that use a PCIC database (deselect with '-m "not crmpdb"')
```

Comment:
This is a confusing name for this marker. the PDP has two databases, `crmpdb` (weather station data) and `pcic_meta` (netCDF data). If I was writing a new test using the database to do some netCDF stuff, I probably wouldn't think to mark it `crmpdb`, since the test has nothing to do with `crmpdb`. Could you name it something that refers to both databases like like `database` or `needsdb` or something?

Reply:
Or I guess you could document it somehow so developers knew.

Reply:
Hm, I think probably all the tests that use the `pcic_meta` database are already marked `bulk_data`, which means they require datafiles on `/storage`. So perhaps instead of extending the `crmpdb` tag to a bunch of tests that don't use that database, you could just exclude the tests tagged `crmpdb` _and_ the tests tagged `bulk_data` when you want to exclude all tests that use a live PCIC database.

Reply:
>  you could just exclude the tests tagged crmpdb and the tests tagged bulk_data when you want to exclude all tests that use a live PCIC database.

I think this is the best way to go about it.